bin/tessel-2.js

```diff
@@ -11,18 +11,58 @@ var nameOption = {
   help: 'The name of the tessel on which the command will be executed'
 };
 
-parser.command('provision')
+var language = {
+  metavar : 'LANG',
+  abbr: 'l',
+  help : 'The language to use <javascript|rust|python|js|rs|py>. Javascript by default'
+}
+
+
+parser.command('init')
   .callback(function(opts) {
-    controller.provisionTessel(opts)
-      .catch(function(err) {
-        if (err instanceof Error) {
+    init(opts)
+    .then(function(){
+      process.exit(0);
+    })
+    .catch(function (err) {
+      logs.warn(err);
+      process.exit(1);
+    });
+  })
+  .option('interactive', {
+    flag: true,
+    abbr: 'i',
+    help: 'Run in interactive mode'
+  })
+  .option('lang', language)
+  .option('directory', {
+    position: 1,
+    help: 'Provide a directory to initialize your project'
+  })
+  .help('Initialize repository for your Tessel project')
+
+
+parser.command('list')
```

Comment:
Why does this commit edit how the command line parser accepts the `list` command?